src/tr1/game/item_actions/chain_block.c

```diff
@@ -8,20 +8,21 @@
 void ItemAction_ChainBlock(ITEM *item)
 {
     if (g_Config.audio.fix_tihocan_secret_sound) {
-        if (g_FlipTimer == 0) {
+        if (Room_GetFlipTimer() == 0) {
             Sound_Effect(SFX_CHAINBLOCK_FX, nullptr, SPM_NORMAL);
-            g_FlipTimer = 1;
+            Room_SetFlipTimer(1);
             return;
         }
     }
 
-    if (g_FlipTimer == 0) {
+    int32_t flip_timer = Room_GetFlipTimer();
+    if (flip_timer == 0) {
         Sound_Effect(SFX_SECRET, nullptr, SPM_NORMAL);
     }
 
-    g_FlipTimer++;
-    if (g_FlipTimer == 55) {
+    Room_SetFlipTimer(++flip_timer);
```

Comment:
Let's avoid preincrementing the arguments.

Reply:
Fixed, coupled with update below.